apps/astarte_realm_management/lib/astarte_realm_management.ex

```diff
@@ -52,7 +52,8 @@ defmodule Astarte.RealmManagement do
       {Astarte.DataAccess, data_access_opts},
       {Astarte.RPC.AMQP.Server, [amqp_queue: Protocol.amqp_queue(), handler: Handler]},
       {Task.Supervisor, name: Astarte.RealmManagement.DeviceRemoverSupervisor},
-      Astarte.RealmManagement.DeviceRemoval.Scheduler
+      Astarte.RealmManagement.DeviceRemoval.Scheduler,
+      {Astarte.RealmManagement.Repo, xandra_opts}
```

Comment:
Let's move it up the children list, close to Astarte.DataAccess, so that the Repo process is started earlier.